src/format/RuboCop.ts

```diff
@@ -1,82 +1,180 @@
 "use strict";
-import * as fs from 'fs';
-import * as path from 'path';
-import * as cp from 'child_process';
-import * as os from 'os';
 
+import * as cp from "child_process";
```

Comment:
Whitespace changes here. [VSCode's guidelines use tabs](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)

Reply:
I'd be happy to dig up a `prettier` config that matches the VSCode guidelines and start using it for vscode-ruby... Might result in a lot of changes, though. Especially if we decided to just run it in one PR. Separate PR, perhaps?

Reply:
Converting the whole codebase should be a standalone PR. In the mean time...hand format? Sorry but following the VSCode guidelines is in the PR template

Reply:
See the new commit in #297. If that one looks good, let me know and I'll make the same change here.

I re-ran prettier with the following options. We should start using this ASAP and then we could avoid these discussions forever. :)

```
{
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "es5",
  "useTabs": true
}
```

Reply:
Good to know on the prettier options. We can add that at a later time. In the mean time, people will have to continue to suffer through the code style guidelines by hand.